CMakeLists.txt

```diff
@@ -77,14 +103,11 @@ if(BUILD_TESTING)
     enable_testing()
 endif()
 
-# Support RPATH?
-option(ENABLE_RPATH "Enable RPATH for this library" ON)
-mark_as_advanced(ENABLE_RPATH)
+# Enable RPATH support for installed binaries and libraries
 include(AddInstallRPATHSupport)
-add_install_rpath_support(BIN_DIRS "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BINDIR}"
-                          LIB_DIRS "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}"
-                          INSTALL_NAME_DIR "${CMAKE_INSTALL_PREFIX}"
-                          DEPENDS ENABLE_RPATH
+add_install_rpath_support(BIN_DIRS "${CMAKE_INSTALL_FULL_LIBDIR}"
```

Comment:
What's the option to enable/disable install rpath equivalent to `ENABLE_RPATH` ? Even if it is a CMake variable, should we explicitly list as an option as we do for `BUILD_SHARED_LIBS` ?

Reply:
It is `CMAKE_SKIP_RPATH` which automatically pops out at configuration time.
If we make it as an option it would probably be redundand, isn't it @drdanz ?